node/src/actors/chain_manager/actor.rs

```diff
@@ -875,7 +875,7 @@ impl ChainManager {
         self.chain_state.unspent_outputs_pool.visit_with_pkh(
             own_pkh,
             |(output_pointer, (vto, _))| {
```

Comment:
```|(output_pointer, _)|```

Reply:
I'd better not commit on holidays :sweat_smile: